### PR TITLE
Custom preamble

### PR DESCRIPTION
**Pull request by jagot**
I implemented a way of customising the preamble, and also which packages are used.

Furthermore, I added the possibility of specifying custom table alignment. This should maybe be a separate pull request; if this is important, I can refactor.

### CODE REVIEW

File: src/LaTeX.jl [hunk 1]
```diff
@@ -182,12 +195,17 @@ function processitem(p, item::TOC, indent)
     ["\\tableofcontents"]
 end
 
-function processitem(p, item::Abstract, indent)
-    r = Any["\\begin{abstract}"]
-    append!(r, processitem(p, item.content, indent))
-    push!(r, "\\end{abstract}")
+function processitem(p, item::Block, indent; options = [])
```
Comment (TotalVerb) [inline src/LaTeX.jl:198]: Are these `options` passed in from anywhere? It seems a bit inconsistent to me that only this method supports options. Options should perhaps be a part of the `Block` type.

**Comment by TotalVerb** [comment]
@rened Bump. This is a pretty solid PR; it would be good to merge it and tag a new version soon.

**Comment by TotalVerb** [comment]
@rened Bump. Are you still around to maintain this package?

**Comment by rened** [comment]
@TotalVerb Hi, sorry, disappeared for a while. Yes, I am, and will incorporate this in the next days!

**Pull request closed without merging**